### ISSUE TITLE
Last online peoples

### ISSUE THREAD

**Issue by DomeSs97** [DenizShabani/telegramscraper#6]
Hi, is possible to make script with last seen peoples but for last hour, last 12 hours etc? with specific time. thanks lot!

**Comment by ChankitSaini** [comment]
> Hi, is possible to make script with last seen peoples but for last hour, last 12 hours etc? with specific time. thanks lot!

Yeah! It's possible, you just need to filter recently active members, all recently seen members will be scrapped and added to the group.
